cypress/integration/stoplight-project.ts

```diff
@@ -11,7 +11,7 @@ describe('Stoplight component', () => {
   });
 
   it('opens via url', () => {
-    cy.visit('/stoplight-project/reference/todos/todo.v1.yaml/components/schemas/Todos');
+    cy.visit('/stoplight-project/c2NoOjEwNg-todo');
```

Comment:
Wasn't the new URL scheme supposed to be more readable? :trollface:

Reply:
These hashes help point to the right node, which is a different "new URL scheme" to what we did recently making URLs _inside_ OpenAPI nicer. 

Everything on the left of the file: shorter and less redundant.
Everything on the right of the file: less squiggly.

Reply:
But then why did the `/components/schemas/Todos` disappear? @mpodlasin ?

Reply:
I literally have no clue. I was desparate to make the build pass in a reasonable way.

Now we can investigate :joy:

Reply:
I understand that it dissipated because it is represented by node slug. We don't need these information in uri as it is now all within node data.